### ISSUE TITLE
Enhance communication between quadrants

### ISSUE THREAD

**Issue by Kimoriz** [Kimoriz/CrowdSimulation#13]

**Comment by tommasomarzi** [comment]
I took a look at the code you uploaded (in particular to the methods quadrant::tresPass() and quadrant::boidMover()) and it seems that should work, but i haven't tried it yet.
So what do you mean specifically?

**Comment by Kimoriz** [comment]
The problem is that on the border of the quadrants boid does not communicate with the ones of the adiacent quadrants so collisions do not occur, and this is a very big problem cause boids tend to fuse with other and this cause a great number of "fake" collision that really slow down the programme =(

**Comment by Kimoriz** [comment]
Anyway i suggest u to download the programme to test it personally, then we will discuss on it

**Comment by tommasomarzi** [comment]
> Anyway i suggest u to download the programme to test it personally, then we will discuss on it

Yep i found an easier way to implement the new lines in the code (yesterday it gave me several compilation errors).


> The problem is that on the border of the quadrants boid does not communicate with the ones of the adiacent quadrants so collisions do not occur, and this is a very big problem cause boids tend to fuse with other and this cause a great number of "fake" collision that really slow down the programme =(

Maybe a vector of copy of boids which are on the border could be a solution?
In this way may we can avoid redrawn problems and we can handle that type of collision (but actually i don't know how much it will optimize the computations, and i think we'll lose something graphically)
We must analyze in depth this problem.

**Comment by Kimoriz** [comment]
Deep look at things seems interesting
